.github/workflows/release.yml

```diff
@@ -42,10 +42,10 @@ jobs:
           commit_message: 'docs: update CHANGELOG.md for ${{ github.ref_name }}'
           file_pattern: CHANGELOG.md
 
-      - name: Set up JDK 21
+      - name: Set up JDK 17
         uses: actions/setup-java@v4
         with:
-          java-version: '21'
+          java-version: '17'
```

Comment:
If Java 21 is needed for all the test to run, I think it's better if we stuck to this version for the release.